src/App.tsx

```diff
@@ -1,29 +1,50 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [perPage, setPerPage] = useState(5);
+  const [currentPage, setCurrentPage] = useState(1);
+
+  const handleChangeSelect = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    const newPerPage = +event.target.value;
+
+    setPerPage(newPerPage);
+    setCurrentPage(1);
+  };
+
+  const handleChangePage = (page: number) => {
+    setCurrentPage(page);
+  };
```

Comment:
this helper makes no sense. It does nothing but calls setState. You can make it directly